service-tests/github.js

```diff
@@ -15,7 +15,15 @@ module.exports = t;
 
 t.create('License')
   .get('/license/badges/shields.json')
-  .expectJSONTypes(Joi.object().keys({ name: 'license', value: Joi.string() }));
+  .expectJSON({ name: 'license', value: 'Creative Commons Zero v1.0 Universal' });
```

Comment:
This looks a bit ridiculous… how about showing the `spdx_id` instead?

Reply:
This is how is currently works https://img.shields.io/github/license/rettichschnidi/clang-tidy-misra.svg. I listed all licenses supported by GitHub in the second comment ^. Should we change behaviour of existing badges? Maybe we could add a new license badge with SPDX id as a value - users will have a choice.

Reply:
Right. I wasn't critiquing your work, just the way we had these set up. What key and spdx_id gets returned for that NCSA license? I think displaying **NCSA** for that is totally fine, just like **CC0** or even **CC0-1.0** is fine.

In general we should avoid providing configuration, when a good default will suffice, since it works against our goals of consistency and legibility.

Reply:
Key is `ncsa` (in general key is always lower case) and SPDX id is `NCSA`. Personally I prefer SPDX id over key as a shorter name. 
Btw, I didn't feel you was critiquing my work :-)